cogs/gpt_3_commands_and_converser.py

```diff
@@ -195,12 +214,12 @@ async def send_help_text(self, ctx):
             title="GPT3Bot Help", description="The current commands", color=0xC730C7
         )
         embed.add_field(
-            name="/g <prompt>",

```

Comment:
Maybe this is a bit confusing to call it "chat", since it isn't exactly a chat and it's more so just a query, maybe we can keep it as /g, or /query, or /ask-gpt?

Reply:
That's a good suggestion, I think ask is good, so the full command would be /gpt ask

Reply:
Are all the commands forced to have the command group name at first? Like for example do we have to do `/dalle draw` now? Is there any way to make the command just `/ask`?

Reply:
You can tab the command so if you type '/ask' and hit tab it'll change itself to '/gpt ask' and on mobile you just write 'askæ and click it like normal. We could relegate just the system commands to being in a group then keep the rest separate, but add the relevant checks to the 'slash_command' checks decorator. This is mainly what I wanted feedback on. Personally I find it fine, but I also see why it could be cumbersome

Reply:
I see, I also personally find it fine because I use tab a lot myself as sort of an automatic reflex lol. I also think the benefit of these being grouped outweighs the extra word in the command and minor hassle, so I think we should proceed, but just to confirm, is there no other way to group commands without a forced prefix for the group name?

Reply:
Not API wise afaik, there's probably some wizardly ways I'm not aware of to do it. I'm not that knowledgeable on it

Reply:
Got it, we can proceed with this merge then, I like the changes. I’ll test locally tomorrow and merge— away from my computer right now